test/signup.test.js

```diff
@@ -0,0 +1,69 @@
+const chai = require('chai');
+const app = require('../index');
+const request = require('supertest')(app);
+const User = require('../models/User');
+const mocha = require('mocha');
+
+const describe = mocha.describe;
+const beforeAll = mocha.before
+
+const expect = chai.expect; 
+describe('POST requests', function() {
+    beforeEach(function(done){
+        User.remove({}, function(err){
+          done();
+        });
+        
+    });
+     describe('/api/users/', function(){
+        
+        describe('status code', function(){
+            it('should be equal to 201', function(done){
+                request.post('/api/users/')
+                .send({
+                    user: {
+                        email: 'goodemailformat@goodemail.com',
```

Comment:
there's this really nice npm module for having mock data called [faker](https://www.npmjs.com/package/faker) you could use that instead